aries-site/src/examples/templates/code-blocks/DarkCodeTheme.js

```diff
@@ -0,0 +1,125 @@
+export const DarkCodeTheme = {
+    'code[class*="language-"]': {
+        'color': '#f8f8f8',
+        'background': 'none',
+        'textAlign': 'left',
+        'whiteSpace': 'pre',
+        'wordSpacing': 'normal',
+        'wordBreak': 'normal',
+        'wordWrap': 'normal',
+        'lineHeight': '1.5',
+        'MozTabSize': '4',
+        'OTabSize': '4',
+        'tabSize': '4',
+        'WebkitHyphens': 'none',
+        'MozHyphens': 'none',
+        'msHyphens': 'none',
+        'hyphens': 'none',
+    },
+    'pre[class*="language-"]': {
+        'color': '#f8f8f8',
+        'background': '#FFFFFF18',
+        'textAlign': 'left',
+        'whiteSpace': 'pre',
+        'wordSpacing': 'normal',
+        'wordBreak': 'normal',
+        'wordWrap': 'normal',
+        'lineHeight': '1.5',
+        'MozTabSize': '4',
+        'OTabSize': '4',
+        'tabSize': '4',
+        'WebkitHyphens': 'none',
+        'MozHyphens': 'none',
+        'msHyphens': 'none',
+        'hyphens': 'none',
+        'padding': '1em',
+        'margin': '0.5em 0',
+        'overflow': 'auto',
+        'borderRadius': '0.3em',
+    },
+    ':not(pre) > code[class*="language-"]': {
+        'background': '#FFFFFF18',
+        'padding': '0.1em',
+        'borderRadius': '0.3em',
+        'whiteSpace': 'normal',
+    },
+    'class-name': {
+        'color': '#FC5A5A',
+    },
+    'maybe-class-name': {
+        'color': '#F740FF',
+    },
+    'comment': {
+        'color': '#CCCCCC',
+    },
+    'function': {
+        'color': '#FC5A5A',
```

Comment:
I am noticing that our contrast ratios are ~2.5-ish and we should be 4.5 for AA. Is this something we should be considering? We may need to make the background color something like 'background' (#263040) in order to get the contrast we need.